client/my-sites/stats/features/modules/stats-locations/style.scss

```diff
@@ -108,19 +108,19 @@ $locations-horizontal-bar-width: 380px;
 				display: inherit;
 			}
 		}
+
+		.stats__summary--narrow-mobile {
+			.stats-card--column-header {
+				padding: 0;
+			}
+		}
 	}
 
 	.stats-card-skeleton.locations-skeleton {
 		&.stats-card-skeleton--with-hero .stats-card__content {
 			flex-direction: column;
 		}
 	}
-
-	.stats__summary--narrow-mobile {
-		.stats-card--column-header {
-			padding: 0;
-		}
-	}
```

Comment:
I'm a little unclear on why this needed to be moved up, is there a cascade issue or is it unintentional reformatting?